lib/Doctrine/ORM/Configuration.php

```diff
@@ -608,7 +608,7 @@ public function addCustomStringFunction($name, $className)
      * @param string $name
      *
      * @return string|null
-     * @psalm-return ?class-string
+     * @psalm-return ?class-string|callable
```

Comment:
```suggestion
     * @psalm-return class-string|callable|null
```

PHP does not allow `?` in unions and we should not use that syntax in doc blocks either.